pylint/checkers/variables.py

```diff
@@ -175,23 +169,14 @@ class VariableVisitConsumerAction(Enum):
 
     Continue -> continue loop to next consumer
     Return -> return and thereby break the loop
-    Consume -> consume the found nodes (second return value) and return
     """
 
     CONTINUE = 0
     RETURN = 1
-    CONSUME = 2
-
-
-VariableVisitConsumerActionAndOptionalNodesType = Union[
-    Tuple[
-        Union[
-            Literal[VariableVisitConsumerAction.CONTINUE],
-            Literal[VariableVisitConsumerAction.RETURN],
-        ],
-        None,
-    ],
-    Tuple[Literal[VariableVisitConsumerAction.CONSUME], List[nodes.NodeNG]],
+
+
+VariableVisitConsumerActionAndOptionalNodesType = Tuple[
```

Comment:
I wonder if this is necessary. With the simplification wouldn't it be better to use this directly instead of an alias?

Reply:
Yes, I was starting to think that myself.